src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  numOfBooks: [],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.numOfBooks.push(action.payload);
```

Comment:
- [ ] [Optional] Well done on your implementation! It's great to see that you have implemented the book slice. Your attention to maintaining data integrity is commendable.
However, I'd like to suggest a slight improvement that could enhance your code quality. Currently, you are using the `array.push()` method to add new elements to an array. While this approach works well, I recommend considering the use of the `array.concat()` method instead.
The [concat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) method of [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) instances allows you to merge two or more arrays. The key advantage here is that it doesn't modify the existing arrays; instead, it returns a new array that combines the contents of the original arrays. This can help maintain data immutability and prevent unexpected side effects.
Keep up the excellent work, and happy coding! 🌟📝🚀"